var/spack/repos/builtin/packages/mapsplice2/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Mapsplice2(MakefilePackage):
+    """MapSplice is a software for mapping RNA-seq data to reference genome
+       for splice junction discovery that depends only on reference genome,
+       and not on any further annotations."""
+
+    homepage = "http://www.netlab.uky.edu/p/bioinfo/MapSplice2"
+    url = "http://protocols.netlab.uky.edu/~zeng/MapSplice-v2.2.1.zip"
+
+    version(
+        "2.2.1",
+        sha256="4f3c1cb49ba0abcfc952de5946ee0b56db28c51f4f4d4f5abca66b4461ca7d05",
+    )
+
+    patch("Makefile.patch")
+    patch("mapsplice_ebwt.patch")
```

Comment:
Have these patches been contributed upstream?

Reply:
I haven't upstreamed it yet.
However, the last update of mapsplice2 is June 2016, should I upstream it?

Reply:
It's always worth a shot. If they no longer maintain it, they can just ignore you. But hopefully they'll use it.

Reply:
Thank you for your reply.
I agree with your opinion. So I'll try upstream patch files.